npm-packages/convex/src/server/impl/query_impl.ts

```diff
@@ -333,7 +339,7 @@ export class QueryImpl implements Query<GenericTableInfo> {
       return null;
     }
     if (first_two_array.length === 2) {
-      throw new Error(`unique() query returned more than one result: 
+      throw new Error(`unique() query returned more than one result from table ${this.tableName}: 
```

Comment:
@thomasballinger what do you think?
I almost feel like it would be more useful to say "unique() returned more than one result from query on ${this.indexName}"

Reply:
@djbalin what do you think about printing indexName here instead of tableName? It'll give more information.